disco_aws_automation/disco_datapipeline.py

```diff
@@ -174,23 +243,45 @@ def delete(self, pipeline):
             raise DataPipelineStateException("Pipeline must be saved before it can be deleted (but...)")
         self._dp_client.delete_pipeline(pipelineId=pipeline._id)
 
-    def fetch_or_create(self, template_name, pipeline_name, pipeline_description, tags, log_location):
+    def fetch_or_create(self, template_name, pipeline_name, pipeline_description, tags, log_location,
+                        force_update=False, metanetwork=None, availability_zone=None):
```

Comment:
&lt;shrug&gt; we can take it out if it seems like a bad thing to have, it was just pretty cheap to add and it annoyed me that I had to make an arbitrary choice deep inside and had no way to control it from the caller.